docs/data/material/components/autocomplete/autocomplete.md

```diff
@@ -207,6 +207,10 @@ In the event that you need to lock certain tags so that they can't be removed, y
 
 {{"demo": "CheckboxesTags.js"}}
 
+### Infinite scrolling
```

Comment:
This `Infinite scrolling` demo should be outside the `Multiple values` section.

Reply:
Placed inside asyncronous requests